maintainers/maintainer-list.nix

```diff
@@ -10977,6 +10977,11 @@
     name = "Maxim Schuwalow";
     email = "maxim.schuwalow@gmail.com";
   };
+  mschwaig = {
```

Comment:
It would be nice to have another mean of contact

Reply:
Added my email in b124565d16c3fdbe0e1294519012f567a7b9f8f6

Reply:
please squash that commit into the init commit

Reply:
thanks for the feedback. Done: 10292050af13675893c4f6075aa98da150908018